packages/site/src/layout/ComponentView.tsx

```diff
@@ -58,7 +58,11 @@ export const ComponentView = () => {
   }, [code, iframe?.current, iframeMobile?.current, type]);
 
   useEffect(() => {
-    if (type === "web" && !PageMeta?.component?.element) {
+    if (
```

Comment:
This got into a nasty update loop if there was no component (a 404). This prevents the nasty loop.